lesson16/index.js

```diff
@@ -61,23 +268,43 @@ function createTabContents() {
 function appendFragment(values) {
     const fragment = document.createDocumentFragment();
 
-    //記事タイトルの数だけliとaを作成
     let articleTitles = [];
     values.forEach((e) => {
         articleTitles.push(e.title);
     });
 
+    let articleComments = [];
+    values.forEach((e) => {
+        articleComments.push(e.comments);
+    });
```

Comment:
```suggestion
  const articleTitles = values.map(value => value.title);

  const articleComments = values.map(value => value.comments);
```
IMO
You can use map to do the same thing.

Reply:
It's true...! I hadn't noticed it at all. 
I'm glad it's a very clean code.
3bef537